src/maker.rs

```diff
@@ -0,0 +1,394 @@
+#![allow(unused_variables)]
+
+use crate::{
+    bitcoin, dai,
+    network::Order,
+    order::{BtcDaiOrder, Position},
+    rate::Spread,
+    MidMarketRate, OngoingTakers, Rate,
+};
+use std::convert::TryFrom;
+
+#[derive(Debug, PartialEq)]
+pub enum NewOrder {
+    Created(BtcDaiOrder),
```

Comment:
Can you write a comment to hint what other variants you would expect to add later?